packages/generator/src/edmx-to-vdm/v4/operation.ts

```diff
@@ -32,11 +32,11 @@ const extractResponse = (response: string) => `${response}.value`;
 function splitMissingOperation(
```

Comment:
[req] This name is confusing, it sounds like this is about one operation that is split. Suggestion: `splitOperationsByMissingImports()` or similar.